core/templates/pages/admin-page/admin-page.directive.ts

```diff
@@ -91,8 +91,8 @@ angular.module('oppia').directive('adminPage', [
               AdminRouterService.showTab($location.path().replace('/', '#'));
             });
             ctrl.userEmail = '';
-            AdminDataService.getDataAsync().then(function(response) {
-              ctrl.userEmail = response.user_email;
+            UserBackendApiService.getUserInfoAsync().then(function(response) {
```

Comment:
maybe userInfo instead of `response`? As response sounds like a JSON dict whereas this is more of a UserInfo object right?

Reply:
Just for clarification. 
Do we use `response` only for JSON 
and for objects, we use its object name ?